packages/experiments/src/components/UnifiedPicker/UnifiedPicker.tsx

```diff
@@ -351,45 +343,43 @@ export const UnifiedPicker = <T extends {}>(props: IUnifiedPickerProps<T>): JSX.
       onCopy={_onCopy}
     >
       <FocusZone direction={FocusZoneDirection.bidirectional} {...focusZoneProps}>
-        <MarqueeSelection selection={selection} isEnabled={true}>
```

Comment:
Is there anything else this is doing for us that we'll lose if we remove it?

Reply:
also let us document this as a comment here so we know